### DISCUSSION TITLE
v0.9.5

### DISCUSSION THREAD

**Discussion by baffler** [baffler/Transparent-Twitch-Chat-Overlay#204] [General]
## Version 0.9.5 - Changes
* The uninstaller will now remove stored settings/data
* Audio for browser enabled by default
* Audio device selection (for sound clips)
* Volume control for sounds clips
* Can now change the folder for sound clips
_When you choose the folder for sound clips, it will scan the folder for .wav and .mp3 files and add them to the list of Chat Sounds to use for notifications when there's a chat message._
* Added an entry in the context menu to open Dev Tools
* CefSharp (Chromium) updated to 117.2.20

~ Known Issues:
 * Jump lists no longer work if you allow multiple instances (it just launches another instance)
 * Can't login to the popout Twitch, this is caused by Twitch blocking login for unsupported browsers

## Installation and Usage
* You can download the Installer now to make it easier to install the program and it will automatically install the required dependencies (VC++ runtimes).
* "Windows protected your PC" message may appear which is the Windows Defender Smart Screen. You'll need to click "More info" and then "Run anyway". You'll only have to do this the first time you launch it.
* After launching the program for the first time, you'll need to set your twitch channel in the settings. You can click the button at the top-left to open settings, right-click the black border at top, right-click the icon in the taskbar, or use the tray icon
* You can move the application by clicking and dragging the thin black border at top, and you can resize it by clicking and dragging on any part of the border
* To hide the borders, just click the [o] button in the top-right, or right-click the icon in your taskbar

![2018-11-26_18-45-47](https://user-images.githubusercontent.com/1980600/49049166-daf52280-f1ab-11e8-94ed-62568029ea03.png)

<hr /><em>This discussion was created from the release <a href='https://github.com/baffler/Transparent-Twitch-Chat-Overlay/releases/tag/'>v0.9.5</a>.</em>

**Comment by pantabya** [comment]
Installing brought up a blue screen and said "running this software might put your PC at risk" ... probably best to avoid running.

**Reply by Flame6** [reply]
This was addressed in the installation and usage section. Try reading it again.

**Comment by FloraleiBloodwood** [comment]
After I installed v0.9.5 the program won't start at all, no sign of it in task manager either. I've reverted to the previous version and it works fine.

**Reply by OGNetheri** [reply]
I have same issue. 0.9.5 doesn't load up. No issues in install. @baffler

**Comment by DeclanPickle** [comment]
should probably open with borders on everytime cause i cant turn them on now and i cant change where it is and when i reopen it, it starts with borders off and the only way to move it is with borders

**Reply by Flame6** [reply]
lower right corner (next to the clock), right click the icon and hit show borders.  
![image](https://github.com/baffler/Transparent-Twitch-Chat-Overlay/assets/12807505/01a9b058-2275-4f80-bf16-0aa9132c8575)
